### ISSUE TITLE
Update mesos-consul to 0.3.0

### ISSUE THREAD

**Issue by stevendborrelli** [CiscoCloud/microservices-infrastructure#791]
Add support for DiscoveryInfo protobufs, which will allow enhanced service discovery for frameworks like Elasticsearch.

**Comment by siddharthist** [comment]
I'm getting some strange errors on GCE after changing the following line in roles/marathon/defaults/main.yml:
```
mesos_consul_image_tag: v0.3
```
```
[centos@lb-mi-worker-001 ~]$ sudo docker run -it ciscocloud/mesos-consul:v0.3
2015/11/09 20:05:51 Failed to connect to 127.0.0.1:2181: dial tcp 127.0.0.1:2181: connection refused
2015/11/09 20:05:52 Failed to connect to 127.0.0.1:2181: dial tcp 127.0.0.1:2181: connection refused
2015/11/09 20:05:53 Failed to connect to 127.0.0.1:2181: dial tcp 127.0.0.1:2181: connection refused
```
This happens on both control and worker nodes.

**Comment by ChrisAubuchon** [comment]
1. Running it on the command line without any arguments isn't going to work. 
2. To fix the role change all of the `--registry` command line arguments in `roles/marathon/templates/mesos-consul.json.j2` to `--consul`

**Comment by stevendborrelli** [comment]
Fixed with #796
